es6-async-await/await.js

```diff
@@ -0,0 +1,68 @@
+import fetch from './fetch.js';
+
+// Keep track of app load time so each log message can be timed.
+// Log messages should all be approximately 1 second apart.
+const startTime = Date.now();
+const elapsed = () => `${Math.round((Date.now() - startTime) / 1000)}s -`;
+
+async function fetchOnce() {
+  try {
+    const response = await fetch('foo/bar.html');
+    const msg = await response;
+    console.log(elapsed(), 'fetchOnce:', msg);
+  } catch (error) {
+    console.log('Error:', error);
+  }
+}
+
+async function fetchSeveral() {
+  try {
```

Comment:
try/catch is nice in all of these, but the original code didn't have a `.catch` so it isn't necessary to have try/catch here.